EnableDetective.yaml

```diff
@@ -21,7 +21,7 @@ Parameters:
     Description: RoleName to create IAM Role in the administrator account and each member account.
   CreateInstanceRole:
     Type: String
-    Description: Select Yes to create an EC2 instance role that can be attached to an instnace in the Master account which will allow the instance to assume the exection role.  Select No if you plan to run the script locally or are creating the stack in a member account.
+    Description: Select Yes to create an EC2 instance role that can be attached to an instnace in the administrator account which will allow the instance to assume the exection role.  Select No if you plan to run the script locally or are creating the stack in a member account.
```

Comment:
There's another typo in "instnace" here. Would you mind correcting it while you're here?